.github/workflows/python.yml

```diff
@@ -134,7 +134,7 @@ jobs:
 
           # workaround to get mypy to figure out the types it should install
           # note that it does not upgrade already installed packages
-          mypy --config-file mypy.ini . > /dev/null 2> /dev/null || true
+          $(echo "${cmd} ${python_files[@]}") > /dev/null 2> /dev/null || true
```

Comment:
Is it for installing packages only used from executable files without .py suffix?

Reply:
Mainly yes. I also ran into a situation where some python files are detected in hidden folders by `find . -type f -not -path '*/\.git/*' -not -path '*/__pycache__/*' -not -path '*/\.mypy_cache/*'` but are ignored by `mypy`. In this case `if [ ${#python_files[@]} -eq 0 ]; then` is satisfied so the `mypy` command is executed but cannot find any file so `mypy --install-types --non-interactive` fails with the following error:
```
error: Can't determine which types to install with no files to check (and no cache from previous mypy run)
```